src/typeDefs/types.ts

```diff
@@ -25,6 +25,8 @@ export const types = gql`
     user: User!
     event: Event!
     feedbackSubmitted: Boolean!
+    createdAt: DateTime
+    updatedAt: DateTime
```

Comment:
make createdAt and updatedAt non nullable in `type` fields, they will never be null in database, if there has never been an update to a database object after it's creation, its createdAt and updatedAt fields will be equal